sova/cli/commands/briefing.py

```
@@ -0,0 +1,126 @@
+
+import typer
+from rich.console import Console
+
+console = Console(stderr=True)
+
+
+def briefing(
+    project: Annotated[Optional[Path], typer.Option("--project", "-p", help="Project directory.")] = None,
+    since: Annotated[Optional[str], typer.Option("--since", help='Time filter (e.g., "2h", "30m", "1d").')] = None,
+    providers: Annotated[Optional[str], typer.Option("--providers", help="Comma-separated providers.")] = None,
+    output_json: Annotated[bool, typer.Option("--json", help="Output as JSON.")] = False,
```

Ruff (0.16.2)

[warning] Boolean default positional argument in function definition

(FBT002)